### DISCUSSION TITLE
Short space "\," inside markdown environment rendered uncorrectly under MikTeX

### DISCUSSION THREAD

**Discussion by nopria** [Witiko/markdown#224] [Q&A] [answered]
The following MWE (to be compiled with XeLaTeX) is rendered correctly by Overleaf but not under MikTeX:

```
\documentclass{book}

\usepackage{fontspec}

\usepackage{polyglossia}
\setdefaultlanguage{english}

\usepackage[hybrid]{markdown}

\begin{document}

\begin{markdown}
Inside markdown environment 34\,cm is rendered as 34,cm by MikTeX.
\end{markdown}

Outside markdown environment 34\,cm is always rendered correctly without comma.

\end{document}

```

**Comment by Witiko** [comment, marked as answer]
The Markdown package is going to become [CommonMark][7]-compliant in version 3.0.0, see Section 4 of [our latest TUGboat article][1]. In CommonMark, [any punctuation character may be backslash-escaped][2]; this has been supported since [Markdown 2.16.0 from August 27, 2022][3], see also #163. Overleaf uses an early image of TeX Live 2022 that only contains Markdown 2.15.2 from May.

You can ensure that the same version of the Markdown package will be used both on Overleaf and in your local installation of MikTeX by downloading a selected version of the Markdown package from [releases][4] and [installing it next to your document][5]. Since version 2.16.0, you would type `34\,cm` as `34\\,cm` in Markdown. Alternatively, you can use [the new `rawAttribute` option][6] added in Markdown 2.18.0 and type ``` `34\,cm`{=tex} ```. Sorry for the inconvenience.

 [1]: https://www.tug.org/TUGboat/production/43-3/tb135novotny-markdown.pdf
 [2]: https://spec.commonmark.org/0.30/#backslash-escapes
 [3]: https://github.com/Witiko/markdown/releases/tag/2.16.0
 [4]: https://github.com/Witiko/markdown/releases
 [5]: https://witiko.github.io/markdown/#portable-installation
 [6]: https://witiko.github.io/markdown/#option-rawattribute
 [7]: https://commonmark.org/
